datumaro/components/environment.py

```diff
@@ -234,6 +241,12 @@ def make_converter(self, name, *args, **kwargs):
             result = result.convert
         return partial(result, *args, **kwargs)
 
+    def make_generator(self, name, *args, **kwargs):
+        result = self.generators.get(name)
+        if isclass(result):
+            result = result.generate_dataset
+        return partial(result, *args, **kwargs)
+
```

Comment:
```suggestion
```

The function is not needed for new members. Also, it is implemented incorrectly, because `DatasetGenerator.generate_dataset` is not static, but was called directly on the class.

Reply:
> The function is not needed for new members.